recipes/libbsd/all/conanfile.py

```diff
@@ -21,7 +26,6 @@ class LibBsdConan(ConanFile):
         "shared": False,
         "fPIC": True,
     }
-    exports_sources = "patches/**"
 
     _autotools = None
```

Comment:
This is not needed with the new helpers and the generate method

Reply:
As this is used in build() and package(): do you have an example how to handle this?

Reply:
Done